packages/fxa-auth-server/bin/key_server.js

```diff
@@ -32,6 +32,11 @@ async function run(config) {
     stripeHelper = createStripeHelper(log, config);
   }
 
+  const redis = require('../lib/redis')(
```

Comment:
Hmm, we also require & instantiate a redis client in `lib/payments/stripe.js`... I wonder if this would be worth a follow up issue to find where all we use redis and have them share this top-level instance?

Reply:
They use different configs, the most important of which is probably the key prefix string.  I do agree that we should sort that out so they can share an instance.